doc/configoptions.md

```diff
@@ -5,161 +5,102 @@ in `data/*/*.file_list` (see [file list docu](files.md)).
 
 The options here are environment variables used by `mk_image`. They are typically set in `Makefile`.
 
+Notation: in the reference below, the uppercase FOO is the value of the option
+(environment variable) lowercase foo.
 
 ## General options
 
-- cache=1|2|4
+<!--
+use this to find out actual usage:
+  grep bin/mk_image Makefile | tr ' ' '\n' | sort -u
+-->
 
-  cache the rpms we use, it is actually a bitmask; the most useful value is 4, which
-  will cache the unpacked rpm in a separate directory
+- alternatives=1
 
-- debug=ignore
+  perform a fix-up of update-alternatives links (because...? so that...?)
```

Comment:
It was an attempt to solve the update-alternatives problem; it's not used (except in the zenroot image) - instead
the inks are maintained manually iin the *.file_list files.

Reply:
incorporated